lib/rules/require-capitalized-comments.js

```diff
@@ -52,11 +93,38 @@ var assert = require('assert');
 module.exports = function() {};
 
 module.exports.prototype = {
-    configure: function(requireCapitalizedComments) {
+    configure: function(options) {
+        var optionName = this.getOptionName();
+
+        var isObject = typeof options === 'object';
+
         assert(
-            requireCapitalizedComments === true,
-            'requireCapitalizedComments option requires a value of true or should be removed'
+            options === true ||
+            isObject,
+            optionName + ' option requires the value `true` ' +
+            'or an object with String[] `allExcept` property'
         );
+
+        if (isObject) {
+            var exceptions = options.allExcept;
+
+            // verify items in `allExcept` property in object are string values
+            assert(
+                Array.isArray(exceptions) &&
+                options.allExcept.every(function(el) { return typeof el === 'string'; }),
+                'Property `allExcept` in ' + optionName + ' should be an array of strings'
+            );
+
+            this._exceptions = {};
```

Comment:
just declare this at the top of `configure` so you don't have to do `|| {}` later.